Active Work Zones Intent

saratlingamarla opened the pull request:
This pull request contains two new intents:
1. WorkZonesOnMyStreetIntent(grabs user address from session and gives no. of work zones near the user) 
2. WorkZonesOnAnyStreetIntent(grabs address input from utterance and gives no. of work zones near the target address).

Please add the tag "hacktoberfest" to the issue before merging this pull request. Please feel free to suggest any additions to this intent and the utterances

jmartini commented on BostonData/lambda_function/lambda_function.py:
```diff
@@ -230,6 +234,90 @@ def get_trash_day_info(intent, session):
+            zone_str = 'zone'
+            start = 'There is'
+
+        speech_output = start + " " + "".join(record['_full_count']) + " " + "active work" + " " + \
```
I think this would be easier to read with something like:
```
speech_output = " ".join([start, record['_full_count'], "active work", zone_str, "on that street."])

jmartini commented:
No need to commit ```BostonData/lambda_function/lambda_function.zip```.

saratlingamarla commented:
i wanted to delete the zip file from the commit, forgot to do that in the last minute

saratlingamarla commented:
@jmartini I changed the file structure like you said:

1. All Intents in an Intents folder and each intent in its own file
2. renamed docs folder to Speech Assets(seems to be the standard name for this folder)
3. 2 new intents for Active Work Zones

Please take a look and merge the PR
